makefile.shade

```diff
@@ -6,3 +16,62 @@ var AUTHORS='Microsoft'
 
 use-standard-lifecycle
 k-standard-goals
+
+#repack-x86 target='compile'
```

Comment:
Add this bit before checking in: https://github.com/aspnet/RazorTooling/commit/c415aa17adb6f311d0d6eb655098c83d025be703